addon/templates/components/flash-message.hbs

```diff
@@ -1,5 +1,5 @@
 {{#if hasBlock}}
-  {{yield this flash}}
+  {{yield this flash (action 'close')}}
```

Comment:
For consistency, we try to use double quotes on templates. Can you wrap `close` in double quotes?